cmd/run.go

```diff
@@ -46,10 +47,11 @@ A full workflow is capable of running the following steps:
   - Compiling the generated SDKs
 
 ` + "If `speakeasy run` is run without any arguments it will run either the first target in the workflow or the first source in the workflow if there are no other targets or sources, otherwise it will prompt you to select a target or source to run.",
-	PreRun:         getMissingFlagVals,
-	Run:            runFunc,
-	RunInteractive: runInteractive,
-	RequiresAuth:   true,
+	PreRun:           preRun,
+	Run:              runFunc,
+	RunInteractive:   runInteractive,
+	RequiresAuth:     true,
+	UsesWorkflowFile: true,
```

Comment:
I don't totally follow this command config here. Run always uses a workflow file

Reply:
It's basically so that the versioning logic can be re-used by anything that runs based on a workflow file. For example, if we add a `speakeasy source publish` command or something